url-plugins/src/main/java/co/cask/hydrator/plugin/realtime/config/UrlFetchRealtimeSourceConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.realtime.config;
+
+/**
+ * Config class for URL Fetch plugin
+ */
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.plugin.PluginConfig;
+
+/**
+ * Config class for Source.
+ */
+public class UrlFetchRealtimeSourceConfig extends PluginConfig {
+
+  @Name("URL")
+  @Description("The URL to fetch data from.")
+  private String url;
+
+  @Name("Interval")
+  @Description("The amount of time to wait between each poll in seconds.")
+  private long intervalInSeconds;
```

Comment:
seems like it would be a good idea to have a property for headers that should be used for the request.